src/streaming/net/HTTPLoader.js

```diff
@@ -370,14 +426,60 @@ function HTTPLoader(cfg) {
                     requestStartTime = new Date();
                     lastTraceTime = requestStartTime;
                     httpRequests.push(delayedRequest.httpRequest);
-                    loader.load(delayedRequest.httpRequest);
+                    _loadRequest(loader, delayedRequest.httpRequest, delayedRequest.httpResponse);
```

Comment:
Yes, since we already returned a Promise when initially loaded the request while delaying it.
BTW, the returned promise of `HTTPLoader.load()` seems always be ignore